### ISSUE TITLE
Issue in Startpage on Brave

### ISSUE THREAD

**Issue by infokiller** [infokiller/web-search-navigator#160]
I built from source, loaded as unpacked and enabled the setting for startpage, but it doesn't work for me.
I use Brave 1.10.93 based on Chromium 83.0.4103.106 on Ubuntu.

Is there more that I should do?

======================
UPDATE
I see that there's a warning and an error:

**Error**: Uncaught (in promise)
Context
_generated_background_page.html
Stack Trace
webext-dynamic-content-scripts.js:1 (anonymous function)

**Warning**: Unrecognized manifest key 'browser_specific_settings'.

_Originally posted by @LaPeste in https://github.com/infokiller/web-search-navigator/issues/82#issuecomment-647357441_

**Comment by infokiller** [comment]
@LaPeste questions:

- Do you see any related errors in the dev console or in the 
- Does it work in regular Chromium?
- Does it work in Firefox?

Re your earlier question, using `web-ext` will help because it guarantees that the issue is not related to any other extension or setting you have.

**Comment by LaPeste** [comment]
- There is only 1 error in the dev console, but it's always there regardless if web-search-navigator is on or off:
`dsearch?query=ff&cat=web&pl=opensearch:1 Unchecked runtime.lastError: Could not establish connection. Receiving end does not exist.`
- It works in Chromium
- It works in Firefox
- it works with `yarnpkg run web-ext run --source-dir src --start-url 'https://www.startpage.com/do/dsearch?query=d&cat=web&pl=opensearch' --target chromium --chromium-binary brave-browser`

So it's some settings I have / some extension.
I'm gonna try turning off each extension one by one hoping it's an extension. I'll let you know.

**Comment by infokiller** [comment]
Thanks for the detailed debugging @LaPeste 
I'll try to fix it for the other themes

**Comment by infokiller** [comment]
I pushed a fix for this. Closing now, but feel free to reopen if it doesn't work for you.

**Comment by LaPeste** [comment]
Great! I tried and it works for the theme I use. I haven't tried the rest but I assume it's fine.

1 more things that's completely unrelated:
when the search on startpage (sp) loads, the first entry is already selected. I'd assume I can click enter to open it right away, but this won't work. Instead, I need to give an interaction with arrows before I can "enter" any link.
I just checked google search; there I can click enter right after the page has loaded. This is what I was expecting, so I assume it's not by design that on sp it doesn't behave properly.
